go.mod

```diff
@@ -5,8 +5,10 @@ go 1.14
 require (
 	github.com/BurntSushi/toml v0.3.1
 	github.com/alessio/shellescape v1.4.1
+	github.com/docker/distribution v2.7.1+incompatible
```

Comment:
Ben is not fan of importing new things , can we just append docker.io if is a shortname?

Reply:
We actually already have a robust image name parser + normalizer for extracting the tag in our own sources that could easy handle this without an additional dependency 😅
It's slightly more complex than just add docker.io if missing but not much more.

Reply:
I can modify it. :smile_cat: 

The introduction of external dependencies is because I also want to discuss whether we should use external dependencies or continue to use our own implementation.

Reply:
our other dependencies are pretty self contained, mostly things like "is-atty" which has no dependencies itself and is somewhat pointless to reimplement.
this one has a comparatively massive set in comparison https://github.com/distribution/distribution/blob/main/go.mod

we have to take care to keep kind easy to embed in other projects.

Reply:
ok， I will submit the fix today